LitvischenkoYevhen/HW6/HW6_task_2.py

```diff
@@ -0,0 +1,7 @@
+while True:
+    name = input('Enter login:')
```

Comment:
the login must be First, 
so you don't need to use the method lower()

Reply:
I understend. thens